spec/hashie/extensions/indifferent_access_with_rails_hwia_spec.rb

```diff
@@ -35,6 +36,10 @@ class CoercableHash < Hash
     include Hashie::Extensions::MergeInitializer
   end
 
+  class MashWithIndifferentAccess < Hashie::Mash
```

Comment:
I think this should split into a separate file, how about `indifferent_access_with_rais_hwia_spec` and `mash_with_indifferent_access_spec`?